### ISSUE TITLE
Insufficient colour contrast

### ISSUE THREAD

**Issue by husseinalhammad** [email-markup-consortium/emailmarkup.org#10]
@M-J-Robbins highlighted an [issue with colour contrast](https://github.com/email-markup-consortium/email-markup-consortium.github.io/pull/9#pullrequestreview-980807174) in #9:

> One small issue is the contrast of white text on the light green (#3dbdb6) but I'd be happy for us to deal with that later so we can get this out.

**Comment by husseinalhammad** [comment]
@dylanatsmith has also flagged this in https://github.com/email-markup-consortium/email-markup-consortium.github.io/pull/13#issuecomment-1137092054

**Comment by dylanatsmith** [comment]
Hadn't seen this issue at the time. 👍🏻

**Comment by husseinalhammad** [comment]
@megbosh can you please help us figure out a different colour combination here?

**Comment by megbosh** [comment]
Here's a screenshot of the page with a new background color. The bg color is `#E9EAF7` and then we'd change the text color on those backgrounds to match the other dark text, and swap the white logo at the top to the full color version. Thoughts?

![screencapture-emailmarkup-org-2022-06-03-15_15_47](https://user-images.githubusercontent.com/59982423/171933833-bb1579c5-82eb-4f62-ab40-aac666023c0f.png)

**Comment by M-J-Robbins** [comment]
Yeah looks good to me, although I did a quick look on the site with dev tools and `#E9EAF7` looks more purple on my screen but a bit more blue on your screenshot. Bit strange, not sure why that's happening.
Could be my eyes are off a little.
<img width="600" alt="EMC home page with new colours" src="https://user-images.githubusercontent.com/7871206/172132323-dd78d33f-0ba3-479b-81df-9f662f08300d.png">

**Comment by megbosh** [comment]
Oh yeah, that's interesting. Do you want to keep the color as is or look for a different one? Both shades work, it could be a difference in monitor set up?

**Comment by M-J-Robbins** [comment]
I'm happy with the colour change, whichever way it comes out.
